doc/notification.markdown

```diff
@@ -191,7 +191,8 @@ Provide one of the following values:
 
   * `10` - The push notification is sent to the device immediately. (Default)
     > The push notification must trigger an alert, sound, or badge on the device. It is an error to use this priority for a push notification that contains only the `content-available` key.
-  * `5` - The push message is sent at a time that conserves power on the device receiving it.
```

Comment:
Are we repeating the Apple push docs here or is that all specific to the package?

Reply:
This is repeating the apple docs. Many of the properties in this doc are repeating